svthalia/concrexit
# Fix bootstrap <select> checkmark repeating

Closes #2602 

### How to test
Steps to test the changes you made:
1. Edit profile so it doesn't validate, but with a correct country.
2. See a single checkmark in the country select box
